snap/validate.go

```diff
@@ -282,8 +280,7 @@ func validateSocketAddrPath(socket *SocketInfo, fieldName string, path string) e
 }
 
 func validateSocketAddrAbstract(socket *SocketInfo, fieldName string, path string) error {
-	// TODO parallel-install: use of proper instance/store name, discuss socket activation in parallel install world
-	prefix := fmt.Sprintf("@snap.%s.", socket.App.Snap.InstanceName())
+	prefix := fmt.Sprintf("@snap.%s.", socket.App.Snap.SnapName())
```

Comment:
Why this change?

Reply:
I should probably add a comment here. The `path` comes from `snap.yaml`, so it can only match snap name (`foo`), but not the actual instance name (`foo_bar`). There will be an obvious conflict in the system if there is more than one snap with the same abstract socket address, but we expect the adminstrator to update the `snap.*.socket` files appropriately.

Reply:
Right, I see. Yes, I think a comment here makes sense.